manual_test.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env node */
+
+const utils = require("./test/utils");
+// const webdriver = require("selenium-webdriver");
```

Comment:
Correct, this is not needed as I moved all of the `webdriver` references to `utils.js`. I think that this is probably better to keep the `button_test.js` as clear and clean as possible.